### PR TITLE
EJ2-Documentation-865442-Ug-Documentation-For-Grid-Scrolling

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

File: ej2-javascript/grid/scrolling/scrolling.md [hunk 1]
```diff
@@ -11,17 +11,19 @@ domainurl: ##DomainURL##
 
 # Scrolling in ##Platform_Name## Grid control
 
- The scrollbar will be displayed in the grid when content exceeds the element [`width`](../api/grid/#width) or [`height`](../api/grid/#height). The vertical and horizontal scrollbars will be displayed based on the following criteria:
+The scrolling feature in the ##Platform_Name## Grid control allows you to navigate through the content that extends beyond the visible area of the grid . It provides scrollbars that are automatically displayed when the content exceeds the specified `width` or `height` of the grid element. This feature is useful when you have a large amount of data or when the content needs to be displayed within a limited space. The vertical and horizontal scrollbars will be displayed based on the following criteria:
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/scrolling/scrolling.md:14]: place the dot once the text gets ended. remove unwanted space.

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

File: ej2-javascript/grid/scrolling/scrolling.md (hunk 1 above)
Comment (harinikrh) [reply ej2-javascript/grid/scrolling/scrolling.md]: removed unwanted spaces

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

**Pull request merged**